sdk/ml/azure-ai-ml/azure/ai/ml/.bumpversion.cfg

```diff
@@ -0,0 +1,3 @@
+[bumpversion]
```

Comment:
I don't think this file is actually part of the package, right?

Reply:
It's gone.